newIDE/app/src/ObjectEditor/Editors/Cube3DEditor.js

```diff
@@ -196,6 +200,34 @@ const Cube3DEditor = ({
             />
           ))}
         </SelectField>
+        <SelectField
+          value={properties.get('materialType').getValue()}
+          floatingLabelText={properties.get('materialType').getLabel()}
+          helperMarkdownText={properties.get('materialType').getDescription()}
+          onChange={(event, index, newValue) => {
+            onChangeProperty('materialType', newValue);
+          }}
+        >
+          <SelectOption
```

Comment:
For this editor:

- Apply basic material (do not react to light, fastest)
- Apply realistic material, without metalness (react to light, good for most rendering)